packages/fx-core/tests/core/FxCore.create.test.ts

```diff
@@ -30,6 +30,16 @@ describe("Core basic APIs for v3", () => {
   beforeEach(() => {
     sandbox.restore();
     setTools(tools);
+    sandbox.stub(environmentManager, "listRemoteEnvConfigs").resolves(ok(["dev"]));
```

Comment:
It looks there are a lot of mock http request/response in these test cases. I am think if below way is more simple and clear to test?
1. Generator has its own unit test case to cover.
2. FxCore test case focus more on create flow, and generator can be considered as a dependent library. Is it possible to mock generator API in this case, which keep the same test coverage but also simplify the test case?

Reply:
Thanks, updated.